docs/CONTRIBUTING.md

```diff
@@ -694,15 +696,18 @@ orchestration. One can pass following environment variables to
 
 One doesn't need to deal with these options directly, there is an _sbt command_
 to orchestrate them all:
+-->
 
 ```bash
 sbt:enso> runProjectManagerDistribution
 ```
 
+<!--
 The above command invokes `buildProjectManagerDistribution`,
 `buildEngineDistribution` and then defines `ENSO_ENGINE_PATH` to connect them
 together and also specifies the `ENSO_JVM_PATH` to the JVM `sbt` process runs
 on.
+-->
```

Comment:
Why not simply delete it if it's irrelevant? We also have history to recover.

Reply:
It is not irrelevant. It works, but it is not relevant to target audience of CONTRIBUTOR.md readers. I don't know where else to put it, so I just commented it out for now.